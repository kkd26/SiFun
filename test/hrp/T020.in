(fn pair => (fn x1 => (fn x2 => (fn x3 => (fn x4 => (fn x5 => x5 (fn y => y)) (fn y => x4 (x4 y))) (fn y => x3 (x3 y))) (fn y => x2 (x2 y))) (fn y => x1 (x1 y))) (fn y => pair y y) ) (fn x => fn y => fn z => z x y)