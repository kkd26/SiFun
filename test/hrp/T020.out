(f60 -> f60 -> f57) -> f57